platforms/common/src/CommonKernels.cpp

```diff
@@ -1080,7 +1080,7 @@ void CommonCalcCMAPTorsionForceKernel::copyParametersToContext(ContextImpl& cont
 class CommonCalcCustomExternalForceKernel::ForceInfo : public ComputeForceInfo {
 public:
     ForceInfo(const CustomExternalForce& force, int numParticles) : force(force), indices(numParticles, -1) {
-        vector<double> params;
+        thread_local static vector<double> params;
```

Comment:
This is in a constructor.  It's only executed once.  There's no benefit to making it static.

Reply:
Thanks for spotting this - removed in the latest commit